Code/GraphMol/Kekulize.cpp

```diff
@@ -460,7 +460,12 @@ void kekulizeFused(RWMol &mol, const VECT_INT_VECT &arings,
     // number of backTracks) and we still need to backtrack
     // can't kekulize this thing
     std::ostringstream errout;
-    errout << "Can't kekulize mol " << std::endl;
+    errout << "Can't kekulize mol.";
+    errout << "  Atoms remaining:";
+    for (unsigned int i = 0; i < nats; ++i) {
+      if (dBndCands[i]) errout << " " << i;
```

Comment:
I find the "atoms remaining" confusing, are these the unkekulized atoms?

I almost would like using _supplementalSmilesLabel to to actually indicate the atoms in the smiles string.